pkg/deploy/generator/resources_rp.go

```diff
@@ -504,7 +499,25 @@ firewall-cmd --add-port=444/tcp --permanent
 firewall-cmd --add-port=445/tcp --permanent
 firewall-cmd --add-port=2222/tcp --permanent
 
-cat >/etc/td-agent-bit/td-agent-bit.conf <<'EOF'
+export AZURE_CLOUD_NAME=$AZURECLOUDNAME
+
+az login -i
+az account set -s "$SUBSCRIPTIONID"
+
+systemctl start docker.service
+az acr login --name "$(sed -e 's|.*/||' <<<"$ACRRESOURCEID")"
+
+MDMIMAGE="${RPIMAGE%%/*}/${MDMIMAGE##*/}"
+docker pull "$MDMIMAGE"
+docker pull "$RPIMAGE"
```

Comment:
Why we are doing this and how this worked before to RPIMAGE?

Reply:
What do you mean? We had this line before (just in different place).

Reply:
sorry, missed it somehow. And I was looking for it but still missed :D